src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,181 @@
+
+  /*
+   * State setup
+   */
+  const [workflow, setWorkflow] = useState({})
+  const [showLog, setShowLog] = useState(false)
+
+  const signal = Utils.useCancellation()
+  const stateRefreshTimer = useRef()
+
+  /*
+   * Data fetchers
+   */
+
+  Utils.useOnMount(() => {
+    const loadWorkflow = async () => {
+      const includeKey = [
+        'end', 'executionStatus', 'failures', 'start', 'status', 'submittedFiles:workflow', 'workflowLog', 'workflowName'
+      ]
+      const wf = await Ajax(signal).Workspaces.workspace(namespace, name).submission(submissionId).getWorkflow(workflowId, includeKey)
+      setWorkflow(wf)
+
+      if (_.includes(wf.status, ['Running', 'Submitted'])) {
+        stateRefreshTimer.current = setTimeout(loadWorkflow, 60000)
+      }
+    }
+
+    loadWorkflow()
+    return () => {
+      clearTimeout(stateRefreshTimer.current)
+    }
+  })
+
+  /*
+   * Page render
+   */
+  const { end, failures, start, status, workflowLog, workflowName, submittedFiles: { workflow: wdl } = {} } = workflow
+
+  const restructureFailures = failuresArray => {
+    const filtered = _.filter(({ message }) => !_.isEmpty(message) && !message.startsWith('Will not start job'), failuresArray)
+    const sizeDiff = failuresArray.length - filtered.length
+    const newMessage = sizeDiff > 0 ? [{ message: `${sizeDiff} jobs were queued in Cromwell but never sent to the cloud backend due to failures elsewhere in the workflow` }] : []
+    const simplifiedFailures = [...filtered, ...newMessage]
+
+    return _.map(({ message, causedBy }) => ({
+      message,
+      ...(!_.isEmpty(causedBy) ? { causedBy: restructureFailures(causedBy) } : {})
+    }), simplifiedFailures)
+  }
+
+  return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
+    workflowDetailsBreadcrumbSubtitle(namespace, name, submissionId, workflowId),
+    _.isEmpty(workflow) ? centeredSpinner() : div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      makeSection('Workflow Status', [
+        div({ style: { lineHeight: '24px' } }, [makeStatusLine(style => statusIcon(status, style), status)])
+      ]),
+      makeSection('Workflow Timing', [
+        div({ style: { marginTop: '0.5rem', display: 'grid', gridTemplateColumns: 'auto 1fr', gap: '0.5rem' } }, [
+          div({ style: styles.sectionTableLabel }, ['Start:']), div([start ? Utils.makeCompleteDate(start) : 'N/A']),
+          div({ style: styles.sectionTableLabel }, ['End:']), div([end ? Utils.makeCompleteDate(end) : 'N/A'])
+        ])
+      ]),
+      makeSection('Links', [
+        div({ style: { display: 'flex', marginTop: '0.5rem' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+            style: { display: 'flex' },
+            tooltip: 'Job Manager'
+          }, [icon('tasks', { size: 18 }), span([' Job Manager'])]),
```

Comment:
```suggestion
          }, [icon('tasks', { size: 18 }), ' Job Manager']),
```

Reply:
(Generally with these I'd tell you to add marginRight to the icon rather than putting a space in the string, since spaces can often be collapsed when rendered, but it's not a huge deal.)